lib/services/nativescript-project-plugins-service.ts

```diff
@@ -0,0 +1,472 @@
+///<reference path="../.d.ts"/>
+"use strict";
+
+import * as path from "path";
+import * as util from "util";
+import {EOL} from "os";
+import Future = require("fibers/future");
+import * as shelljs from "shelljs";
+import * as semver from "semver";
+
+import PluginsDataLib = require("../plugins-data");
```

Comment:
Only the `MarketplacePluginData` class is used from this file and can be captured as follows:
```Typescript 
import {MarketplacePluginData} from "../plugins-data";
```